cirq-google/cirq_google/engine/calibration_to_noise_properties.py

```diff
@@ -37,6 +37,8 @@
 if TYPE_CHECKING:
     import cirq
 
+_ZPhaseData = Dict[str, Dict[str, Dict[Tuple[ops.Qid, ...], float]]]
```

Comment:
Optional:  Do you think it is better to call this _ZPhaseDataType?

Also, it might be good to document what the keys of this dict of dict of dict is.

Reply:
I am not sure.  That class has some extra stuff like an associated EngineJob and Calibration which may not be relevant.  I like the idea of using a typed class rather than a complicated nest of dictionaries though.

Reply:
Tried making this a class, but it didn't help much (mostly ended up being a wrapper around the same nest of dicts). Opted instead to move the type to util, add a docstring, and share it between the files in this PR.